doc/API.md

```diff
@@ -76,23 +76,25 @@ Each rule in axe-core has a number of tags. These provide metadata about the rul
-| `ACT`            | W3C approved Accessibility Conformance Testing rules |
-| `section508`     | Old Section 508 rules                                |
-| `section508.*.*` | Requirement in old Section 508                       |
-| `TTv5`           | Trusted Tester v5 rules                              |
-| `TT*.*`          | Test ID in Trusted Tester                            |
-| `experimental`   | Cutting-edge rules, disabled by default              |
-| `cat.*`          | Category mappings used by Deque (see below)          |
+| Tag Name         | Accessibility Standard / Purpose                                                                                              |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| `wcag2a`         | WCAG 2.0 Level A                                                                                                              |
+| `wcag2aa`        | WCAG 2.0 Level AA                                                                                                             |
+| `wcag2aaa`       | WCAG 2.0 Level AAA                                                                                                            |
+| `wcag21a`        | WCAG 2.1 Level A                                                                                                              |
+| `wcag21aa`       | WCAG 2.1 Level AA                                                                                                             |
+| `wcag22aa`       | WCAG 2.2 Level AA                                                                                                             |
+| `best-practice`  | Common accessibility best practices                                                                                           |
+| `wcag***`        | WCAG success criterion e.g. wcag111 maps to SC 1.1.1                                                                          |
+| `ACT`            | W3C approved Accessibility Conformance Testing rules                                                                          |
+| `section508`     | Old Section 508 rules                                                                                                         |
+| `section508.*.*` | Requirement in old Section 508                                                                                                |
+| `TTv5`           | Trusted Tester v5 rules                                                                                                       |
+| `TT*.*`          | Test ID in Trusted Tester                                                                                                     |
+| `EN.301.549`     | Rule required under [EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf) |
```

Comment:
The `.` separators here feel a bit inconsistent with other tags as a separator for what are spaces in the official standard being referenced. Other existing cases either omit a separator (`Section 508` -> `section508`) or use a hyphen (`best-bractice`). I think `EN-301-549` is the nicest-looking consistent option, but I think any of `EN301549`, `en301549`, `en-301-549` would be fine.

This feedback is minor; I don't think creates big backcompat issues, so if anyone else felt strongly about the `.`s, I can live with the inconsistency.

Reply:
Dashes make sense. @straker what do you think of `EN-301-549`?

Reply:
I'm good with dashes